frontend/src/components/PlayerList/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Player from "./Player";
+import styles from "./style.module.css";
+
+interface PlayerListProps {
+  gameState: "lobby" | "midround" | "endround" | "endgame";
+}
+
+const PlayerList = ({ gameState }: PlayerListProps) => {
+  let showScores: boolean = true;
+
+  if (gameState === "lobby") {
+    showScores = false;
+  }
+
+  // TODO: Retrieve players from backend
+  const me: string = "AlexVerkerk";
+
+  const dummyPlayers: {
```

Comment:
Did you intend to set this component up to use Context after this PR?

Reply:
Yeah, the idea is that when our context providers are set up it'll be `const players = useContext(...)`. Similarly, `me` could be retrived from localStorage or wherever we keep that.

Reply:
Cool sounds good!